ports/mimxrt10xx/boards/metro_m7_1011/flash_config.c

```diff
@@ -14,14 +14,14 @@ __attribute__((section(".boot_hdr.conf")))
 const flexspi_nor_config_t qspiflash_config = {
     .pageSize = 256u,
     .sectorSize = 4u * 1024u,
-    .ipcmdSerialClkFreq = kFLEXSPISerialClk_30MHz,
+    .ipcmdSerialClkFreq = kFLEXSPISerialClk_133MHz,
     .blockSize = 0x00010000,
     .isUniformBlockSize = false,
     .memConfig =
     {
         .tag = FLEXSPI_CFG_BLK_TAG,
         .version = FLEXSPI_CFG_BLK_VERSION,
-        .readSampleClkSrc = kFLEXSPIReadSampleClk_LoopbackFromDqsPad,
+        .readSampleClkSrc = kFLEXSPIReadSampleClk_LoopbackFromSckPad,
```

Comment:
what does this change _mean_? It's nice, if it enables the flash to be clocked higher.

Reply:
It means we can use/reset the DQS pad. (Otherwise flash stops working when you reset DQS even when not connected externally.)